### PR TITLE
Support flavor on the web using flutter `dart-define`

### PR DESCRIPTION
**Pull request by armanddidierjean**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,11 @@ NB: a trailing slash is required.
...
 
 ### Linting
+
 Titan support linting according to the official [Flutter static analysis options](https://dart.dev/guides/language/analysis-options).
 
 The linter can be launched using:
+
 ```
 dart format .
 flutter analyze --fix
```
Comment (maximeComeOn) [inline README.md:39]: It is `dart fix --apply`and we should apply it before the formating to ensure formatting is correct event after applying changes

Comment (maximeComeOn) [reply README.md:39]: We can also run `dart fix --dry-run` to visualize the change that would be done if the run was with `--apply`

Comment (maximeroucher) [inline README.md:39]: It is dart fix --applyand we should apply it before the formating to ensure formatting is correct event after applying changes

We can also run dart fix --dry-run to visualize the change that would be done if the run was with --apply

**Review by julien4215 — APPROVED** [review]

**Review by guillaumetavernier — APPROVED** [review]

**Review by julien4215 — CHANGES_REQUESTED** [review]
We need to keep defining two different environments for alpha and prod because we should not use the same value for `PLAUSIBLE_HOST` and `PLAUSIBLE_DOMAIN` in alpha or production.

**Review by julien4215 — APPROVED** [review]

**Pull request merged**